runtime/bindings/python/iree/runtime/build_requirements.txt

```diff
@@ -6,5 +6,5 @@
 numpy>=1.19.4
 pybind11>=2.8.0
 requests
```

Comment:
Also adds the version constraints for requests?

Reply:
For reasons that aren't clear to me, the containers weren't actually updated. I'm confused, but built again and they seem to be behaving as expected now.